administrator/components/com_installer/src/Model/DiscoverModel.php

```diff
@@ -305,4 +305,24 @@ protected function getEmptyStateQuery()
 
 		return $query;
 	}
+
+	/**
+	 * Checks uninstalled extensions in extensions table.
```

Comment:
```suggestion
	 * Checks for not installed extensions in extensions table.
```

i appreeciate this was wrong before your pr